helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml

```
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: prometheus-agent.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: prometheus-agent
+    rules:
+    ## Page Atlas if prometheus agent fails to send samples to MC prometheus.
+    - alert: PrometheusAgentRemoteWriteNotWorking
+      annotations:
+        description: '{{`Prometheus agent remote write is failing.`}}'
+        summary: Prometheus agent fails to send samples to remote write endpoint.
+        opsrecipe: prometheus-agent-remote-write-failed/
+      expr: absent_over_time(up{instance="prometheus-agent",cluster_type="workload_cluster"}[10m])
```

Not needed with the label added at the top
```suggestion
      expr: absent_over_time(up{instance="prometheus-agent"}[10m])
```